src/countOccurrences.js

```diff
@@ -18,7 +18,17 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let count = 0;
+
+  const checkPhrase = phrase.toLowerCase();
```

Comment:
what does check mean?
```suggestion
  const phraseLowerCase = phrase.toLowerCase();
```